backends/intel_gpu/kernels/elementwise_kernel.cc

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dnn_support.hpp"
+#include "paddle/phi/capi/all.h"
+#include "phi_funcs.h"
+
+
+namespace custom_kernel {
+
+template <typename T>
+void MultiplyRawKernel(const phi::Context& dev_ctx,
```

Comment:
this is pure C++ code on host, and no use in intel_gpu device kernel. Please clean and remove

Reply:
fixed